core/commands/pin.go

```diff
@@ -94,9 +94,9 @@ on disk.
 
 var rmPinCmd = &cmds.Command{
 	Helptext: cmds.HelpText{
-		Tagline: "Unpin an object from local storage",
+		Tagline: "Recursively unpin an object from local storage",
 		ShortDescription: `
-Removes the pin from the given object allowing it to be garbage
+Recursively removes the pin from the given object allowing it to be garbage
```

Comment:
the description + tagline should still say:

```
"Unpin an object...
"Removes the pin...
```
but can just add: 
```
(by default, recursively. use -r=false for direct pins)
```
or something